### ISSUE TITLE
Images name to train

### ISSUE THREAD

**Issue by vikbeo** [skotz/captcha-breaking-library#10]
Hi Scott, during the training I see that: 
Image name will be the value in captcha but if there is no other way to name the file we will not able to train multiple image with same value but different in format. Say I have image: ABC.bmp & if another image name ABC.bmp with different format, I will not able to train the solver with 2nd one if not overwrite 1st.

Do you see any way we can solve this?

Thank you.

**Comment by skotz** [comment]
Good idea. I hadn't thought of that before. This isn't currently supported, but I'll tag this as an enhancement and perhaps add that functionality sometime.

**Comment by vikbeo** [comment]
Amazing support! I do see you may already got the idea. Just advise my idea, it may only need to pickup captcha's value from first path of filename such as: abc123_timestamp.bmp
Get only value till reach first underscore. Leave the remain part for differentiate with other variants.

Another thing that you may want to update the guide that in order to train the solver the image should be in bmp format otherwise the trainer will not find any image and no trainer folder made.
